Standard.AI.OpenAI.Tests.Unit/Services/Orchestrations/AIFiles/AIFileOrchestrationServiceTests.cs

```diff
@@ -46,8 +46,8 @@ public static TheoryData DependencyValidationExceptions()
             {
                 new LocalFileValidationException(someInnerException),
                 new LocalFileDependencyValidationException(someInnerException),
-                new AIFileValidationException(someInnerException),
-                new AIFileDependencyValidationException(someInnerException)
+                new AIFileValidationException(message: "AI file validation error occurred, fix errors and try again.", someInnerException),
+                new AIFileDependencyValidationException(message: "AI file dependency validation error occurred, contact support.", someInnerException)
```

Comment:
These lines exceed the 120 character line length limit, when they are broken then they need to be seperated by an extra line.

```suggestion

                new AIFileValidationException(
                    message: "AI file validation error occurred, fix errors and try again.",
                    innerException: someInnerException),
                
                new AIFileDependencyValidationException(
                    message: "AI file dependency validation error occurred, contact support.",
                    innerException: someInnerException)
```